layer.yaml

```diff
@@ -3,6 +3,6 @@ includes: ['layer:basic']  # if you use any interfaces, add them here
 options:
   basic:
     packages:
-      - git
       - build-essential
-      - software-properties-common
```

Comment:
justification on removing this: it provides add-apt-repository, which isn't used in this layer.